docs/installation.md

```diff
@@ -30,7 +38,12 @@ pip install poetry
 poetry install
 ```
 
-This will create a virtual environment for you and install all developer and docs dependencies within it.
+!!! info
+
+    If you wish to use the package's jupyter integrations, run `poetry install -E jupyter` instead.
```

Comment:
@dexter2206 as far as I understand `poetry install --with jupyter` is preferable syntax. Could you explain whether that's correct and why actually?